### ISSUE TITLE
Experiment Reproducing Problems

### ISSUE THREAD

**Issue by 01rice20** [ci4r/CI4R-Activity-Recognition-datasets#4]
Hello, I was trying to reproduce the experiment of VGG-16 and CAE Classification (Which are corresponding to the Table 1 & 3 Results), and I ran into two problems as follows:

1.  VGG-16's score just cannot reach Table 1
I followed the file "vggNet-24_gridsearch.ipynb" and the parameters are as the paper written, but the accuracy is less about 5-9%, and  are always different everytime I train. So I was wondering if the dataset setting is correct. All I did is follow the "create_dataset.ipyhb" file, is there anything I should notice?

2. I can't find the json and weight file using in "Transfer_Learning_CAE.ipynb", where can I get them to reproduce the score in Table 3?

Looking forward to your reply, thank you!!

**Comment by ci4r** [comment]
Hi @01rice20 ,

A grid search is applied on the vggnet to find the optimal hyperparameter selection, and in the original uploaded ipynb file, I can see that the accuracy reaches to those values. In order to get the same results during the training process, you need to fix all of your seeds, you can take a look at [this tutorial](https://machinelearningmastery.com/reproducible-results-neural-networks-keras/).

Weights files are large and cannot be uploaded to github directly. I uploaded the CAE training script, `Conv Autoencoder.ipynb`. It trains a CAE and saves the weights and the model json file. I hope this helps.

**Comment by 01rice20** [comment]
Hello, 
Many thanks to your reply, but I still got some confusion here:

(1) Dataset
I found 2 versions of the spectrogram training data inside the Google Cloud dataset link, they're store in `/Spectrograms/ and /Raw Data/raw_data.zip/Spectograms_77_24_Xethrue/` respectively, and I don't know which of them is the correct dataset that corresponding to `Conv Autoencoder.ipynb, Transfer_Learning_CAE.ipynb, Transfer_Learning_CNN.ipynb`, the numbers of the dataset shows in the cell output are not matching with both of the datasets.

(2) VGG-16
I can see the cell outputs in `vggNet-24_gridsearch.ipynb` have reach the accuracy shows in the paper of Tabel 1. But according to the paper, I saw "batch size of 32, 40 epochs, learning rate of 1e-4, momentum of 0.9 and Adam optimizer were set as the training parameters". So I was wondering if the score in Table 1 is grid search or the parameters I've mentioned above?

(3) CAE
So if I want to reproduce the score show in Table 3, 4. I should train the "Train for all datasets" cell in `Conv Autoencoder.ipynb` , after getting the weight and json files, train `Transfer_Learning_CAE.ipynb`. Is my understanding correct?
